client/src/components/App/Content/LandingPage/InvestigationTable/TableFilter/TableFilterStyles.ts

```diff
@@ -10,26 +10,53 @@ const useStyles = makeStyles((theme: Theme) => ({
         marginBottom: theme.spacing(1),
         borderRadius: 16,
     },
+    startCard: {
+        display: 'flex',
+        justifyContent: 'flex-start',
+        alignItems: 'center',
+    },
+    endCard: {
+        display: 'flex',
+        justifyContent: 'flex-end',
+        alignItems: 'center',
+    },
```

Comment:
consider using material UI's [Box ](https://material-ui.com/system/flexbox/)or [Grid](https://material-ui.com/components/grid/) components , or at least create a class that contains display:flex alignItems:center and then another class for different justify options

Reply:
It's already a grid component that acts as a flexbox because of the specific design that was needed

Reply:
if it's a grid than you dont need display:'flex' , you can use <Grid justify='X' alignItems='Y' />

Reply:
your comment is cut in the middle